### PR TITLE
feat: ani-sync on windows

### PR DESCRIPTION
**Pull request by iamchokerman**
# Pull Request Template

## Type of change

- [ ] Bug fix
- [x] Feature
- [x] Documentation update

## Description

Tested with a Windows User, works as expected

## Checklist

- [X] any anime playing
- [X] bumped version
- [X] next, prev and replay work
- [X] quality works
- [X] downloads work
- [X] quality works with downloads
- [X] select episode -a and rapid resume work
- [X] syncplay -s works
- [X] autoplay, aka range selection, works

## Additional Testcases

- The safe bet: One Piece
- Episode 0: Saenai Heroine no Sodatekata ♭
- Unicode: Saenai Heroine no Sodatekata ♭
- Not uploaded: one piece dub episode 590
- Unreleased: soredemo ayumu wa yosetekuru
- Short id (for decryption): Log Horizon episode 1-2

### CODE REVIEW

File: ani-cli [hunk 1]
```diff
@@ -37,13 +37,14 @@ help_text () {
 	  ${0##*/} -h | -D | -U | -V
 
 	Options:
-	  -c continue watching anime from history
+	  -v use VLC as the media player
+	  -q set video quality (best|worst|360|480|720|1080)
+		-s watch anime together with friends, using Syncplay (works with mpv only)
```
Comment (Derisis13) [inline ani-cli:42]: Ehm

File: ani-cli [hunk 2]
```diff
@@ -521,6 +522,8 @@ while getopts 'svq:dp:chDUVa:' OPT; do
 		s)
 			case "$(uname -s)" in
 				Darwin*) player_fn="/Applications/Syncplay.app/Contents/MacOS/syncplay" ;;
+				MINGW*) player_fn="/c/Program Files (x86)/Syncplay/Syncplay.exe" ;;
+				*Msys) player_fn="/c/Program Files (x86)/Syncplay/Syncplay.exe" ;;
```
Comment (Derisis13) [inline ani-cli:526]: These can be consolidated to one line, tags separated with a `|`

File: ani-cli (hunk 1 above)
Comment (Derisis13) [reply ani-cli:42]: (Probably some whitespace error here, can't see it clear since on mobile)

**Review by Derisis13 — CHANGES_REQUESTED** [review]
This shall make a nice release once complete

**Comment by port19x** [comment]
> This shall make a nice release once complete

definitely

**Review by Derisis13 — APPROVED** [review]
I don't have anyone to watch together though...

**Comment by port19x** [comment]
I'd like to merge 707 first and then release this

**Comment by iamchokerman** [comment]
sounds good

File: README.md [hunk 3]
```diff
@@ -19,7 +19,7 @@
 </p>
 
 <h3 align="center">
-A cli to browse and watch anime. This tool scrapes the site <a href="https://gogoplay5.com">gogoplay.</a>
+A cli to browse and watch anime (alone AND with friends). This tool scrapes the site <a href="https://gogoplay5.com">gogoplay.</a>
```
Comment (port19x) [inline README.md:22]: Very big brain marketing, I like it

Comment (iamchokerman) [reply README.md:22]: :)

**Comment by port19x** [comment]
Release checklist

- [ ] Matrix and Revolt links (in #707)
- [ ] Repology badge (in #707)
- [ ] #707 merged
- [ ] Fix long access time (issue #709)
- [ ] Update Termux instructions to advise usage of native package

**Comment by port19x** [comment]
We're very close to release. If possible I'd like to merge this this evening

**Comment by port19x** [comment]
> ~Wait wait.. we don't hv native package on termux yet..~ ~I also read all those link u send but right now it's not in standard repos..~ I checked it now and installed it.. but it's installing the standard mpv... Not the custom one we wrote in our documentation Also it's an older version.. like 2.1.0 and not working..

Which is why I currently have an open PR at the termux-packages repo...

**Comment by port19x** [comment]
I just want to see if termux native package is viable under ideal conditions

**Pull request merged**